components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2371,7 +2378,15 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         request.setReturnImmediately(true);
 
         // this triggers the snapshots, but returns early! cmp. waitForSnapshot to wait for it's completion
-        const resp = await client.takeSnapshot(ctx, request);
+        const resp = await client.takeSnapshot(ctx, request).catch((err) => {
```

Comment:
Unsure about the syntax here. The `await` will cause the `takeSnapshot` to throw in case of error, so the `catch` handling should really be in a `try/catch` block rather than in the promise chain

Reply:
Good point, updated